test/integration/cookies.js

```diff
@@ -56,10 +60,16 @@ describe('Cookies', () => {
                 const rawCookies = response.headers['set-cookie'];
                 cookie = getCasCookieValue(rawCookies[0]);
                 credentials = response.body.data[0];
-                return;
-            });
+                cookieUtils = require('../../lib/utils/cas-cookie-utils.js');
+
+                return cookieUtils.verifyAndParse(cookie,
+                    server.plugins['hapi-redis'].client);
+            }).then((parsed) => {
+                parsedCookie = parsed;
+            }).catch((err) =>{console.dir(err); return err.data;});
```

Comment:
by catching the error, I think that you might be forcing the mocha task to pass. 

Consider removing the catch statement all together, or re-throwing the error inside of it.